style.css

```diff
@@ -545,3 +559,330 @@ button.footer-b:active {
   background: #0d151e;
   border-radius: 10px;
 }
+
+@media (min-width: 768px) {
+  nav {
+    display: flex;
+    align-items: center;
+    height: 40px;
+    background: #3c3a39;
+    overflow: hidden;
+  }
+
```

Comment:
# Optional

- [ ] Let justify these link to left `text-align:left;` to match the design.(line 414)
![Screenshot from 2022-06-02 13-58-58](https://user-images.githubusercontent.com/65860678/171625038-d447c8d3-0c67-493f-a5f6-0f6b3e7fd976.png)